### ISSUE TITLE
TRY100: Detect all try except not just generic ones

### ISSUE THREAD

**Issue by ehooo** [tonybaloney/pycharm-security#80]
**Describe the bug**
The insecure issue is only the generic exceptions cause the flow must regiter unknown flows.
In the following case and/or other similars cases, `pass` or `continue` is not insecure cause is an expected situation.

**To Reproduce**
Code:
```
PROTO_SSL_VERSION = (
    (ssl.PROTOCOL_TLSv1, 'v1'),
    (ssl.PROTOCOL_SSLv23, 'v2.3'),
)
try:
    PROTO_SSL_VERSION.append((ssl.PROTOCOL_SSLv2, 'v2'))
except AttributeError:
    pass  # This protocol is not available if OpenSSL is compiled with the OPENSSL_NO_SSL2 flag.
```

**Expected behavior**
Only the following code must be marked as insecure:
```
try:
   ..code..
except:
  pass or continue
```
or
```
try:
   ..code..
except Exception:
  pass or continue
```

**Additional context**
 - Plugin Version: 1.11.0
 - PyCharm Version (or IDE version): PyCharm 2019.3
 - Python Version: 2.7, 3,7

**Comment by jugmac00** [comment]
Ha... just wanted to post a similar issue:
```
        try:
            os.unlink(self.fileName())
        except OSError:
            pass
```

As I expect the deletion could fail, I think this is no security issue.

**Comment by tonybaloney** [comment]
🤔 this is a good point.
Out of curiosity, did it raise an issue with the `ssl.PROTOCOL_SSLv2` import?

**Comment by ehooo** [comment]
Hi @tonybaloney if the OpenSSL is complited with the proper flag the AttributeError will not be raised. (https://docs.python.org/3/library/ssl.html#ssl.PROTOCOL_SSLv2)

There is no rule that detect thee SSLv2, but this is not the ticket issue, it was related to "try pass" and "try continue" rule.

By the way, do you have some link to try to help you coding (debugging, making plugins for PyCharm, ... whatever)? I never use write kotlin code.

**Comment by tonybaloney** [comment]
Released in 1.12.0
